### ISSUE TITLE
add genome_description upon asset pull

### ISSUE THREAD

**Issue by nsheff** [refgenie/refgenconf#99]
Raised by @natefoo

When the first asset for a genome is pulled, we fix the genome_digest, but not the genome_description. The genome description is in fact never populated into the local config it seems, though it can be retrieved via API (eg http://refgenomes.databio.org/v2/genome/hg38).

It would be nice to populate the genome description as well, just so users have that info in their config file.

**Comment by natefoo** [comment]
@stolarczyk thanks! Is it possible with f121abcb0f57c9f318ecf00815a8fe453b5f5929 to update an existing local config? Or do I need to re-fetch assets?

**Comment by stolarczyk** [comment]
the descriptions are populated in `RefGenConf.pull`, there is no dedicated method to pull _just_ the descriptions as I don't expect people fetching just these once this fix is released. Do you think that would be useful?
However, one could write a Python script to populate the descriptions in an existing config, using the existing API. For example for `hg38` sth like:

```python
import requests
import refgenconf
resp = requests.get("http://refgenomes.databio.org/v2/genome/hg38")
genome_attrs = resp.json()
rgc = refgenconf.RefGenConf("/pth/to/cfg.yml")
rgc.update_genomes(genome="hg38", data=genome_attrs)
```

**Comment by natefoo** [comment]
Thanks, that'll work fine for my purposes.

**Comment by natefoo** [comment]
@stolarczyk when I attempt to use this with refgenconf pulled just now from the dev branch on github I get:

```pytb
Traceback (most recent call last):
  File "./update_description.py", line 15, in <module>
    rgc = refgenconf.RefGenConf("genomes_config.yaml")
  File "/home/refgenie/refgenie/lib64/python3.6/site-packages/refgenconf/refgenconf.py", line 120, in __init__
    raise ConfigNotCompliantError(msg)
refgenconf.exceptions.ConfigNotCompliantError: This genome config (v0.3) is not compliant with v0.4 standards. To use it, please downgrade refgenie: 'pip install refgenie==0.7.0'. For config format documentation please see http://refgenie.databio.org/en/latest/genome_config/
```

Is there a way to version migrate my config?

**Comment by stolarczyk** [comment]
ohh, so we're currently working on a major config revamp on dev: instead of genome names we use fasta digests and let users set aliases. As of now there's no version migrate feature, we might add it just before the release. 

This fix is independent of the big changes in the works, so I `cherry-pick`ed it and and added to a new branch: bugfix. This is compliant with the current config format. Could you use this branch for now and let us know if you encounter any other problems? Once you confirm everything's fine we can make a small release!

Sorry for the inconvenience.

**Comment by natefoo** [comment]
That worked, thanks!

**Comment by stolarczyk** [comment]
released in v0.9.2
